CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+### main
+
+* Bump Maps SDK to 11.5.0.
```

Comment:
since iOS and Android version are different, let's specify it here?

Reply:
Specified it in this PR https://github.com/mapbox/mapbox-maps-flutter/pull/615